src/Models/Coupon.php

```diff
@@ -82,6 +83,26 @@ public function isNotExpired(): bool
         return ! static::$bindable->isExpired();
     }
 
+    /**
+     * Check if code is enabled.
+     *
+     * @return bool
+     */
+    public function isEnabled(): bool
+    {
+        return $this->is_enable;
```

Comment:
There should be a `getIsEnabledColumn()` method in the `DefinesColumns` trait to make the column name overrideable.

```suggestion
        return $this->{static::$bindable->getIsEnabledColumn()};
```